src/lime/protocol/constants/common_constants.py

```diff
@@ -0,0 +1,7 @@
+class CommonConstants:
+    """Commom constants used in project."""
+
+    DOMAIN = 'take.net'
```

Comment:
DOMAIN and NAME are constants for testing, right? You don't need to put them here, these are the module constants only